src/main/java/com/poiji/util/Casting.java

```diff
@@ -4,9 +4,9 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.Objects;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
```

Comment:
I would just import only the required classes.

Reply:
I will consider importing only the required classes @samhalpa , thanks!